src/App.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import { getProducts } from "./api/api";
+import Header from "./component/Header";
+import AllProduct from "./component/AllProduct";
+import BestProduct from "./component/BestProduct";
+// import Market from "./pages/Market";
+// import Dropdown from "./component/Dropdown";
+// import SortIcon from "./images/ic_sort.svg";
+// import SearchIcon from "./images/ic_search.svg";
+
+const LIMIT = 6;
+
+function App() {
+  const [order, setOrder] = useState('createAt');
+  const [items, setItems] = useState([]);
+  const [bestItem, setBestItem] = useState([]);
+  const [offset, setOffset] = useState(0);
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+
+  const sortedItems = [...items].sort((a, b) => b[order] - a[order]);
+  const BestProductItem = bestItem.slice(0, 4);
```

Comment:
일반적으로 컴포넌트 외의 변수에는 PascalCase를 사용하지 않는 게 좋습니다!
그리고 배열 데이터를 가졌음을 알 수 있게 네이밍해 주세요.
(+handleLoad 리뷰 참고해 주세요)
```suggestion
  const bestProductItems = items.slice(0,4);
```